backend/corpora/dataset_processing/process.py

```diff
@@ -133,157 +133,19 @@
 
 """
 
-import logging
 import os
-import subprocess
 import sys
-import typing
-from datetime import datetime
-from os.path import join
-
-import numpy
-import scanpy
-
-from backend.corpora.common.corpora_orm import (
-    ConversionStatus,
-    DatasetArtifactFileType,
-    ProcessingStatus,
-    ValidationStatus,
-    UploadStatus,
-)
-from backend.corpora.common.entities import Dataset, DatasetAsset
-from backend.corpora.common.utils.db_helpers import processing_status_updater
+
```

Comment:
- deleted `create_artifacts` and `process` 
- Moved a bunch of functions closer to where they are used files.